.github/workflows/ci-runbook.yml

```diff
@@ -0,0 +1,23 @@
+name: 'CI - Runbook'
+on:
+  pull_request:
+    paths:
+      - runbooks
+
+jobs:
+  build-runbook-readme:
+    runs-on: ubuntu-20.04
+    defaults:
+      run:
+        working-directory: runbooks
+    steps:
+      - uses: actions/checkout@v2
+      - run: |
+          curl https://raw.githubusercontent.com/ekalinin/github-markdown-toc/0.8.0/gh-md-toc -o gh-md-toc
```

Comment:
@StewGoin Thoughts on inclusion of this third-party dependency here?

Reply:
It's fine w/ the same discussion had with @keithdadkins on the parallel PR for it regarding making sure we keep the dependency up-to-date and hygienic and such.